Auto update failing for zoxide

<br>Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the zoxide package, but it failed. 

Here's the output of the update script:

<details><summary>Output log</summary>
<pre lang="bash">

INFO: Updating zoxide [Current version: 0.8.0]...
INFO: package being updated to 0.8.1.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   136  100   136    0     0   2229      0 --:--:-- --:--:-- --:--:--  2229

100  128k    0  128k    0     0   741k      0 --:--:-- --:--:-- --:--:--  741k

You are about to commit these changes:

--------------------
diff --git a/packages/zoxide/build.sh b/packages/zoxide/build.sh
index 5c266cd77..29dad7ac8 100644
--- a/packages/zoxide/build.sh
+++ b/packages/zoxide/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/ajeetdsouza/zoxide
 TERMUX_PKG_DESCRIPTION="A faster way to navigate your filesystem"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.8.0
+TERMUX_PKG_VERSION="0.8.1"
 TERMUX_PKG_SRCURL=https://github.com/ajeetdsouza/zoxide/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=111c5f1cd92b1cb54e2f0a801003098a601c653ec5a378f3d0ea1c9659854477
+TERMUX_PKG_SHA256=55f4bf374421f842f5541d95db14d7a63f5d24e269613fb95f769f00cdb9ffc3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
--------------------

zoxide: update to 0.8.1


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'termux/package-builder'...
Creating new container...
Unable to find image 'termux/package-builder:latest' locally
latest: Pulling from termux/package-builder
6eb9a17f92a2: Pulling fs layer
1c7a5dfb0b91: Pulling fs layer
a2db41113a1b: Pulling fs layer
400985b64afa: Pulling fs layer
400985b64afa: Waiting
1c7a5dfb0b91: Download complete
400985b64afa: Verifying Checksum
400985b64afa: Download complete
6eb9a17f92a2: Verifying Checksum
6eb9a17f92a2: Download complete
6eb9a17f92a2: Pull complete
1c7a5dfb0b91: Pull complete
a2db41113a1b: Verifying Checksum
a2db41113a1b: Download complete
a2db41113a1b: Pull complete
400985b64afa: Pull complete
Digest: sha256:b682b729992003d128b44c44a0cb6d931142045e120e019fb3d705d027c394ed
Status: Downloaded newer image for termux/package-builder:latest
badf6caabd43903d70fd208f5babfae4ebd18ba177415ce48af8f4d2ccd7fc2f
Changed builder uid/gid... (this may take a while)
gpg: key B0076E490B71616B: 10 signatures not checked due to missing keys
gpg: key B0076E490B71616B: public key "Henrik Grimler <henrik@grimler.se>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found
gpg (GnuPG) 2.2.20; Copyright (C) 2020 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

gpg: key 5A897D96E57CF20C: 4 signatures not checked due to missing keys
gpg: key 5A897D96E57CF20C: public key "Termux Releases (Termux automatic builds) <contact@termux.com>" imported
gpg: key 1B388B024A48460E: public key "Termux Releases (Termux automatic builds) <contact@termux.org>" imported
gpg: Total number processed: 2
gpg:               imported: 2
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   2  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: depth: 1  valid:   2  signed:   0  trust: 2-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg (GnuPG) 2.2.20; Copyright (C) 2020 Free Software Foundation, Inc.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

termux - building zoxide for arch aarch64...
Downloading https://raw.githubusercontent.com/termux/termux-elf-cleaner/v1.10/termux-elf-cleaner.cpp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  7569  100  7569    0     0  87000      0 --:--:-- --:--:-- --:--:-- 87000
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:24: warning: "DT_AARCH64_BTI_PLT" redefined
   24 | #define DT_AARCH64_BTI_PLT 0x70000001
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2875: note: this is the location of the previous definition
 2875 | #define DT_AARCH64_BTI_PLT      (DT_LOPROC + 1)
      | 
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:25: warning: "DT_AARCH64_PAC_PLT" redefined
   25 | #define DT_AARCH64_PAC_PLT 0x70000003
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2876: note: this is the location of the previous definition
 2876 | #define DT_AARCH64_PAC_PLT      (DT_LOPROC + 3)
      | 
/home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:26: warning: "DT_AARCH64_VARIANT_PCS" redefined
   26 | #define DT_AARCH64_VARIANT_PCS 0x70000005
      | 
In file included from /home/builder/.termux-build/_cache/termux-elf-cleaner.cpp:16:
/usr/include/elf.h:2877: note: this is the location of the previous definition
 2877 | #define DT_AARCH64_VARIANT_PCS  (DT_LOPROC + 5)
      | 
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  26320      0 --:--:-- --:--:-- --:--:-- 26320
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3442      0 --:--:-- --:--:-- --:--:--  3442
gpg: Signature made Sun 24 Apr 2022 08:59:44 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   1  trust: 0-, 0q, 0n, 0m, 0f, 2u
gpg: depth: 1  valid:   1  signed:   0  trust: 1-, 0q, 0n, 0m, 0f, 0u
gpg: next trustdb check due at 2024-08-25
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 72 1271k   72  927k    0     0   831k      0  0:00:01  0:00:01 --:--:--  831k
100 1271k  100 1271k    0     0  1132k      0  0:00:01  0:00:01 --:--:-- 1131k
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  27750      0 --:--:-- --:--:-- --:--:-- 27750
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2125      0 --:--:-- --:--:-- --:--:--  2125
gpg: Signature made Fri 15 Apr 2022 10:57:29 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/stable/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 46344  100 46344    0     0   212k      0 --:--:-- --:--:-- --:--:--  211k
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  44324      0 --:--:-- --:--:-- --:--:-- 44324
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   7059      0 --:--:-- --:--:-- --:--:--  7059
gpg: Signature made Sun 24 Apr 2022 03:10:58 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 82  296k   82  243k    0     0   745k      0 --:--:-- --:--:-- --:--:--  742k
100  296k  100  296k    0     0   904k      0 --:--:-- --:--:-- --:--:--  901k
Downloading https://github.com/ajeetdsouza/zoxide/archive/v0.8.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   136  100   136    0     0   1700      0 --:--:-- --:--:-- --:--:--  1700

100  128k    0  128k    0     0   675k      0 --:--:-- --:--:-- --:--:--  675k
Applying ndk-patch: bits-struct_file.h.patch
Applying ndk-patch: dirent.h.patch
Applying ndk-patch: grp.h.patch
Applying ndk-patch: langinfo.h.patch
Applying ndk-patch: linux-fcntl.h.patch
Applying ndk-patch: paths.h.patch
Applying ndk-patch: pwd.h.patch
Applying ndk-patch: redefine-TCSAFLUSH.patch
Applying ndk-patch: semaphore.h.patch
Applying ndk-patch: stdio.h.patch
Applying ndk-patch: stdlib.h.patch
Applying ndk-patch: sys-cdefs.h.patch
Applying ndk-patch: syslog.patch
Applying ndk-patch: unistd.h.patch
Applying ndk-patch: utmp.h.patch
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/21/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/22/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/23/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/24/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/26/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/27/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/28/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/29/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/30/libz.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libaaudio.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libamidi.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libandroid.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libbinder_ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libcamera2ndk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libc.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libdl.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libEGL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv1_CM.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv2.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libGLESv3.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libicu.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libjnigraphics.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/liblog.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libmediandk.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libm.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativehelper.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libnativewindow.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libneuralnetworks.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenMAXAL.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libOpenSLES.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libstdc++.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libsync.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libvulkan.so'
termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/31/libz.so'
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: syncing channel updates for '1.60.0-x86_64-unknown-linux-gnu'
info: latest update on 2022-04-07, rust version 1.60.0 (7737e0b5c 2022-04-04)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'

info: default toolchain set to '1.60.0-x86_64-unknown-linux-gnu'
  1.60.0-x86_64-unknown-linux-gnu installed - rustc 1.60.0 (7737e0b5c 2022-04-04)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source $HOME/.cargo/env
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
  Installing zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)
    Updating crates.io index
 Downloading crates ...
  Downloaded atty v0.2.14
  Downloaded hashbrown v0.11.2
  Downloaded lazy_static v1.4.0
  Downloaded memchr v2.4.1
  Downloaded mime_guess v2.0.4
  Downloaded unicode-xid v0.2.2
  Downloaded version_check v0.9.4
  Downloaded textwrap v0.15.0
  Downloaded syn v1.0.91
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded nix v0.24.1
  Downloaded dirs-sys v0.3.7
  Downloaded ordered-float v2.10.0
  Downloaded nom v7.1.1
  Downloaded proc-macro2 v1.0.37
  Downloaded dirs v4.0.0
  Downloaded clap v3.1.12
  Downloaded clap_complete v3.1.2
  Downloaded libc v0.2.124
  Downloaded clap_complete_fig v3.1.5
  Downloaded askama_shared v0.12.2
  Downloaded askama_derive v0.11.2
  Downloaded minimal-lexical v0.2.1
  Downloaded indexmap v1.8.1
  Downloaded heck v0.4.0
  Downloaded fastrand v1.7.0
  Downloaded dunce v1.0.2
  Downloaded clap_derive v3.1.7
  Downloaded clap_lex v0.1.1
  Downloaded bincode v1.3.3
  Downloaded autocfg v1.1.0
  Downloaded askama_escape v0.10.3
  Downloaded os_str_bytes v6.0.0
  Downloaded num-traits v0.2.14
  Downloaded mime v0.3.16
  Downloaded memoffset v0.6.5
  Downloaded glob v0.3.0
  Downloaded cfg-if v1.0.0
  Downloaded bitflags v1.3.2
  Downloaded askama v0.11.1
  Downloaded quote v1.0.18
  Downloaded proc-macro-error v1.0.4
  Downloaded unicase v2.6.0
  Downloaded termcolor v1.1.3
  Downloaded strsim v0.10.0
  Downloaded serde_derive v1.0.136
  Downloaded serde v1.0.136
  Downloaded anyhow v1.0.57
   Compiling version_check v0.9.4
   Compiling proc-macro2 v1.0.37
   Compiling autocfg v1.1.0
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.91
   Compiling memchr v2.4.1
   Compiling libc v0.2.124
   Compiling heck v0.4.0
   Compiling os_str_bytes v6.0.0
   Compiling hashbrown v0.11.2
   Compiling lazy_static v1.4.0
   Compiling strsim v0.10.0
   Compiling termcolor v1.1.3
   Compiling textwrap v0.15.0
   Compiling bitflags v1.3.2
   Compiling serde_derive v1.0.136
   Compiling mime v0.3.16
   Compiling minimal-lexical v0.2.1
   Compiling askama_escape v0.10.3
   Compiling serde v1.0.136
   Compiling anyhow v1.0.57
   Compiling cfg-if v1.0.0
   Compiling fastrand v1.7.0
   Compiling glob v0.3.0
   Compiling dunce v1.0.2
   Compiling unicase v2.6.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling indexmap v1.8.1
   Compiling memoffset v0.6.5
   Compiling num-traits v0.2.14
   Compiling clap_lex v0.1.1
   Compiling quote v1.0.18
   Compiling nom v7.1.1
   Compiling atty v0.2.14
   Compiling dirs-sys v0.3.7
   Compiling mime_guess v2.0.4
   Compiling nix v0.24.1
   Compiling ordered-float v2.10.0
   Compiling dirs v4.0.0
   Compiling clap_derive v3.1.7
   Compiling clap v3.1.12
   Compiling askama_shared v0.12.2
   Compiling clap_complete v3.1.2
   Compiling askama_derive v0.11.2
   Compiling bincode v1.3.3
   Compiling clap_complete_fig v3.1.5
   Compiling askama v0.11.1
   Compiling zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)
    Finished release [optimized] target(s) in 1m 31s
  Installing /data/data/com.termux/files/usr/bin/zoxide
   Installed package `zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)` (executable `zoxide`)
warning: be sure to add `/data/data/com.termux/files/usr/bin` to your PATH to be able to run the installed binaries
cp: -r not specified; omitting directory '/home/builder/.termux-build/zoxide/src/man/man1'
ERROR: failed to build.
</pre>
</details>

Thanks!

<i>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/2215207793">2215207793</a><br>
Timestamp: 2022-04-24 10:25:42 UTC<br>
</i><br><h4>EDIT:</h4><br>It failed again.

Here's the output of the update script:

<details><summary>Output log</summary>
<pre lang="bash">

INFO: Updating zoxide [Current version: 0.8.0]...
INFO: package being updated to 0.8.1.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   136  100   136    0     0   2125      0 --:--:-- --:--:-- --:--:--  2158

100  128k    0  128k    0     0   768k      0 --:--:-- --:--:-- --:--:--  768k

You are about to commit these changes:

--------------------
diff --git a/packages/zoxide/build.sh b/packages/zoxide/build.sh
index 5c266cd77..29dad7ac8 100644
--- a/packages/zoxide/build.sh
+++ b/packages/zoxide/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/ajeetdsouza/zoxide
 TERMUX_PKG_DESCRIPTION="A faster way to navigate your filesystem"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.8.0
+TERMUX_PKG_VERSION="0.8.1"
 TERMUX_PKG_SRCURL=https://github.com/ajeetdsouza/zoxide/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=111c5f1cd92b1cb54e2f0a801003098a601c653ec5a378f3d0ea1c9659854477
+TERMUX_PKG_SHA256=55f4bf374421f842f5541d95db14d7a63f5d24e269613fb95f769f00cdb9ffc3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
--------------------

zoxide: update to 0.8.1


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'termux/package-builder'...
termux - building zoxide for arch aarch64...
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   116k      0 --:--:-- --:--:-- --:--:--  116k
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3062      0 --:--:-- --:--:-- --:--:--  3062
gpg: Signature made Sun 24 Apr 2022 12:19:13 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1268k  100 1268k    0     0  2984k      0 --:--:-- --:--:-- --:--:-- 2977k
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  85935      0 --:--:-- --:--:-- --:--:-- 85935
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   6941      0 --:--:-- --:--:-- --:--:--  6941
gpg: Signature made Fri 15 Apr 2022 10:57:29 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0   125k      0 --:--:-- --:--:-- --:--:--  125k
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   8330      0 --:--:-- --:--:-- --:--:--  8330
gpg: Signature made Sun 24 Apr 2022 12:08:17 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://github.com/ajeetdsouza/zoxide/archive/v0.8.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   136  100   136    0     0   4533      0 --:--:-- --:--:-- --:--:--  4533

100  128k    0  128k    0     0  1296k      0 --:--:-- --:--:-- --:--:-- 1296k
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: syncing channel updates for '1.60.0-x86_64-unknown-linux-gnu'
info: latest update on 2022-04-07, rust version 1.60.0 (7737e0b5c 2022-04-04)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'

info: default toolchain set to '1.60.0-x86_64-unknown-linux-gnu'
  1.60.0-x86_64-unknown-linux-gnu installed - rustc 1.60.0 (7737e0b5c 2022-04-04)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source $HOME/.cargo/env
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: installing component 'rust-std' for 'aarch64-linux-android'
  Installing zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)
    Updating crates.io index
 Downloading crates ...
  Downloaded autocfg v1.1.0
  Downloaded bitflags v1.3.2
  Downloaded askama_escape v0.10.3
  Downloaded askama_shared v0.12.2
  Downloaded clap_complete v3.1.2
  Downloaded fastrand v1.7.0
  Downloaded clap_derive v3.1.7
  Downloaded lazy_static v1.4.0
  Downloaded memoffset v0.6.5
  Downloaded minimal-lexical v0.2.1
  Downloaded version_check v0.9.4
  Downloaded cfg-if v1.0.0
  Downloaded mime v0.3.16
  Downloaded os_str_bytes v6.0.0
  Downloaded quote v1.0.18
  Downloaded strsim v0.10.0
  Downloaded termcolor v1.1.3
  Downloaded syn v1.0.91
  Downloaded bincode v1.3.3
  Downloaded dirs-sys v0.3.7
  Downloaded textwrap v0.15.0
  Downloaded nix v0.24.1
  Downloaded mime_guess v2.0.4
  Downloaded libc v0.2.124
  Downloaded askama_derive v0.11.2
  Downloaded dirs v4.0.0
  Downloaded clap_complete_fig v3.1.5
  Downloaded clap v3.1.12
  Downloaded askama v0.11.1
  Downloaded atty v0.2.14
  Downloaded proc-macro2 v1.0.37
  Downloaded ordered-float v2.10.0
  Downloaded heck v0.4.0
  Downloaded glob v0.3.0
  Downloaded dunce v1.0.2
  Downloaded clap_lex v0.1.1
  Downloaded anyhow v1.0.57
  Downloaded unicode-xid v0.2.2
  Downloaded unicase v2.6.0
  Downloaded serde_derive v1.0.136
  Downloaded serde v1.0.136
  Downloaded proc-macro-error-attr v1.0.4
  Downloaded proc-macro-error v1.0.4
  Downloaded num-traits v0.2.14
  Downloaded nom v7.1.1
  Downloaded memchr v2.4.1
  Downloaded indexmap v1.8.1
  Downloaded hashbrown v0.11.2
   Compiling version_check v0.9.4
   Compiling proc-macro2 v1.0.37
   Compiling autocfg v1.1.0
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.91
   Compiling memchr v2.4.1
   Compiling libc v0.2.124
   Compiling heck v0.4.0
   Compiling hashbrown v0.11.2
   Compiling os_str_bytes v6.0.0
   Compiling bitflags v1.3.2
   Compiling textwrap v0.15.0
   Compiling termcolor v1.1.3
   Compiling strsim v0.10.0
   Compiling lazy_static v1.4.0
   Compiling serde_derive v1.0.136
   Compiling mime v0.3.16
   Compiling minimal-lexical v0.2.1
   Compiling serde v1.0.136
   Compiling askama_escape v0.10.3
   Compiling anyhow v1.0.57
   Compiling cfg-if v1.0.0
   Compiling fastrand v1.7.0
   Compiling dunce v1.0.2
   Compiling glob v0.3.0
   Compiling unicase v2.6.0
   Compiling proc-macro-error-attr v1.0.4
   Compiling proc-macro-error v1.0.4
   Compiling indexmap v1.8.1
   Compiling num-traits v0.2.14
   Compiling memoffset v0.6.5
   Compiling clap_lex v0.1.1
   Compiling quote v1.0.18
   Compiling nom v7.1.1
   Compiling atty v0.2.14
   Compiling dirs-sys v0.3.7
   Compiling mime_guess v2.0.4
   Compiling nix v0.24.1
   Compiling ordered-float v2.10.0
   Compiling dirs v4.0.0
   Compiling clap_derive v3.1.7
   Compiling askama_shared v0.12.2
   Compiling clap v3.1.12
   Compiling bincode v1.3.3
   Compiling askama_derive v0.11.2
   Compiling clap_complete v3.1.2
   Compiling clap_complete_fig v3.1.5
   Compiling askama v0.11.1
   Compiling zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)
    Finished release [optimized] target(s) in 1m 40s
  Installing /data/data/com.termux/files/usr/bin/zoxide
   Installed package `zoxide v0.8.1 (/home/builder/.termux-build/zoxide/src)` (executable `zoxide`)
warning: be sure to add `/data/data/com.termux/files/usr/bin` to your PATH to be able to run the installed binaries
cp: -r not specified; omitting directory '/home/builder/.termux-build/zoxide/src/man/man1'
ERROR: failed to build.
</pre>
</details>

Thanks!

<i>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/2215511917">2215511917</a><br>
Timestamp: 2022-04-24 12:27:39 UTC<br>
</i>

--- a/packages/zoxide/build.sh
+++ b/packages/zoxide/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/ajeetdsouza/zoxide
 TERMUX_PKG_DESCRIPTION="A faster way to navigate your filesystem"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.8.0
+TERMUX_PKG_VERSION=0.8.1
 TERMUX_PKG_SRCURL=https://github.com/ajeetdsouza/zoxide/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=111c5f1cd92b1cb54e2f0a801003098a601c653ec5a378f3d0ea1c9659854477
+TERMUX_PKG_SHA256=55f4bf374421f842f5541d95db14d7a63f5d24e269613fb95f769f00cdb9ffc3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
@@ -15,5 +15,5 @@ termux_step_pre_configure() {
 termux_step_post_make_install() {
 	# Install man page:
 	mkdir -p $TERMUX_PREFIX/share/man/man1/
-	cp $TERMUX_PKG_SRCDIR/man/* $TERMUX_PREFIX/share/man/man1/
+	cp -r $TERMUX_PKG_SRCDIR/man/* $TERMUX_PREFIX/share/man/man1/
 }

